packages/generate-pdf/test/generate-pdf.spec.ts

```diff
@@ -24,13 +27,84 @@ type TestMap = {
   localOnly?: boolean;
 };
 
+/*
+  When adding a new fixture version, add a new test suite to the testMap below.
+  For unreleased versions, add the localOnly flag. Be sure to update isLatestVersion to the new version.
+  To add more than one fixture per version, use the following naming format:
+  valid_cert_<number>.json
+  valid_cert_<number>.pdf
+  valid_cert_<number>.html
+*/
+
+type CertificateTestMap = {
+  type: SchemaDirUnion;
+  version: string;
+  localOnly?: boolean;
+  isLatestVersion?: boolean;
```

Comment:
As I mention in another comment below, the `localOnly` flag is the one that is used for unreleased versions, the `isLatestVersion` is used to indicate that we should also run the PDF generation with the latest generation script. This particular test is only run on the latest version. Maybe I can add a clearer comment explaining what this property is for? Or maybe rename it something like `isLatestReleasedVersion` or `useLocalGeneratorScript`? But I think either way, a better explanation is needed to avoid this same confusion in the future.

Reply:
Exactly, `isLatestVersion` and `isUnreleasedVersion` are completely unrelated in this case. I like the renaming idea, I'll go with that. I'll also add some better documentation for it for next time.